lib/xcodeproj/constants.rb

```diff
@@ -381,6 +380,7 @@ module Constants
         'CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF'    => 'YES',
         'CLANG_WARN_OBJC_LITERAL_CONVERSION'      => 'YES',
         'CLANG_WARN_OBJC_ROOT_CLASS'              => 'YES_ERROR',
+        'CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER' => 'YES',
```

Comment:
I think this might require an update in cocoapods to turn it off?

Reply:
Should already be done in #9905

Reply:
Yup already done in https://github.com/CocoaPods/CocoaPods/pull/9905

Reply:
sorry I hadn't refreshed before commenting :D